notifications/README.md

```diff
@@ -0,0 +1,58 @@
+# Notifications
+
+## Notification Object
+
+```
+{
+  id: string,
+  username: string,
```

Comment:
Let's use `userId`s as much as possible (because a user can change username in future, but never the userid)